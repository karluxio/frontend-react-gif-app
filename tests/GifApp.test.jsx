import { render, screen } from "@testing-library/react"
import { GifApp } from "../src/GifApp"

describe('testing on <GifApp />', () => {
  test('should match against snapshot', () => {
    const { container } = render(<GifApp />)
    expect(container).toMatchSnapshot()
  })
})  