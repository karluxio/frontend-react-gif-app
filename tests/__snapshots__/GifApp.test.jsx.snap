// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing on <GifApp /> should match against snapshot 1`] = `
<div>
  <h1>
    Gif App
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="Add a category"
      type="text"
      value=""
    />
  </form>
  <h3>
    One Punch
  </h3>
  <h3>
    Loading...
  </h3>
  <div
    class="card-grid"
  />
</div>
`;
